src/components/SQFormScrollableCard/SQFormScrollableCard.tsx

```diff
@@ -56,15 +70,11 @@ export type SQFormScrollableCardProps<Values extends FormikValues> = {
   submitButtonText?: string;
   /** Component to render as the Subheader */
   SubHeaderComponent?: React.ReactNode;
-  /** The Title for the Header component */
-  title?: string;
   /**
    * Yup validation schema shape
    * https://jaredpalmer.com/formik/docs/guides/validation#validationschema
    * */
   validationSchema?: AnyObjectSchema;
-  /** Boolean used to determine if title/header is enabled or disabled */
-  isHeaderDisabled?: boolean;
```

Comment:
Checking my understanding, does this make isHeaderDisabled a required prop?

Reply:
@hvilander That's a great callout and totally my bad!  Thanks for calling this out.